applications/app/services/ApplicationsInteractiveRendering.scala

```diff
@@ -29,13 +29,43 @@ object DotcomRendering extends RenderingTier
  */
 
 object ApplicationsInteractiveRendering {
```

Comment:
Small, but the `Applications` prefix here feels a bit redundant. (Realise not part of these changes though.)

Reply:
Renamed.